language/de/acp/ban.php

```diff
@@ -77,11 +77,11 @@
 	'PERMANENT'		=> 'Dauerhaft',
 
 	'UNTIL'						=> 'Bis zum',
-	'USER_BAN'					=> 'Einen oder mehrere Benutzernamen sperren',
+	'USER_BAN'					=> 'Einen oder mehrere Benutzer mittels Benutzernamen sperren',
```

Comment:
Ich finde in dem Fall ist der Benutzername eigentlich schon ein "Werkzeug". "Benutzername sperren" ist für mich "Benutzername verbieten", also niemand kann sich mit dem Benutzernamen registrieren. Hier wird aber wirklich der Benutzer gesperrt. Die Übersetzung hat mich eigentlich auch schon immer verwirrt.